libqtile/window.py

```diff
@@ -1026,8 +1026,11 @@ def _enablefloating(self, x=None, y=None, w=None, h=None,
             self.height = h
         self._reconfigure_floating(new_float_state=new_float_state)
 
-    def togroup(self, group_name=None):
-        """Move window to a specified group"""
+    def togroup(self, group_name=None, switch_group=False):
```

Comment:
I think it'd be really cool if we could start to put things like this behind keyword-only arguments, now that we are on python 3. That would make the process of deprecating much easier, as we could easily add/remove named arguments.

Reply:
Good to know keyword-only arguments :)

I leave it as-is for `group_name`, as forcing it to keyword-only will break configurations. Also, I think keyword-only arguments is more readable.